src/Symfony/Component/Security/Core/Authentication/Provider/DaoAuthenticationProvider.php

```diff
@@ -56,7 +57,7 @@ protected function checkAuthentication(UserInterface $user, UsernamePasswordToke
         $currentUser = $token->getUser();
         if ($currentUser instanceof UserInterface) {
             if ($currentUser->getPassword() !== $user->getPassword()) {
-                throw new BadCredentialsException('The credentials were changed from another session.');
+                throw new AccountPasswordChangedException('The credentials were changed from another session.');
```

Comment:
As the new Exception extends a different base Exception (account status instead of bad credentials), this is a pretty big BC break